modules/serverless-containersource-custom-sources.adoc

```diff
@@ -0,0 +1,193 @@
+[id="serverless-containersource-custom-sources_{context}"]
+= Creating custom event sources by using a container source
+
+You can use a container source to create and manage a container for your custom event source image.
+
+In order to implement a custom event source by using a container source, you must first create a container image of your event source, and then create a container source that specifies the correct configuration, including the container image URI.
+
+== Guidelines for creating a container image
+
+* The container image can be developed using any language, and can be built and published with any tool you like.
+
+* The main process of the container image must accept parameters from arguments and environment variables.
```

Comment:
lol this was a suggestion from QE. Previously it was `main` method, but @cardil pointed out that since folks can use many different languages to write their source, it's better to refer to it in a more generic way. I would say it's OK as is and will probably make sense to developers, unless there's some other suggestion about how to word it.